egs/mini_an4/asr1/conf/lm_transformer.yaml

```diff
@@ -0,0 +1,24 @@
+# network architecture
+model-module: transformer
+att-unit: 16
+embed-unit: 2
+head: 8
+layer: 2
+pos-enc: none
+unit: 10
+
+# minibatch related
+batchsize: 2
+maxlen: 40
+
+# optimization related
+opt: adam
+schedulers: lr=cosine
+dropout-rate: 0.0
+epoch: 3
+gradclip: 1.0
+lr: 1e-4
+lr-cosine-total: 3
+lr-cosine-warmup: 2
```

Comment:
Let me confirm the usage.
If we specify lr2=cosine, we can specify lr2-cosine-total. Is my understanding correct?

Reply:
yes only if `lr2` is available as optimizer's hyperparam and `total` in the scheduler. so it should be `<optimizer param>-<scheduler-name>-<scheduler-param>`.

Reply:
I will add this to lm_train.py